Bottom: 859ba30ede306025e68c705d14992fc23cd8bb67
Top:    117dd79acfb442f12b0ede88c0d21e92e545a268
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-03-23 00:13:06 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
